src/app/core/core.module.ts

```diff
@@ -25,6 +25,7 @@ import {CourseComponent} from './components/courses/components/course/course.com
   ],
   imports: [
     CommonModule
-  ]
+  ],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
why do you still need CUSTOM_ELEMENT_SCHEMA?

Reply:
deleted